builder/dockerfile/internals.go

```diff
@@ -107,10 +111,17 @@ func (b *Builder) exportImage(state *dispatchState, imageMount *imageMount, runC
 func (b *Builder) performCopy(state *dispatchState, inst copyInstruction) error {
 	srcHash := getSourceHashFromInfos(inst.infos)
 
+	var commentStr string
+	if inst.flChown != "" {
+		commentStr = fmt.Sprintf("%s --chown=%s %s in %s ", inst.cmdName, inst.flChown, srcHash, inst.dest)
+	} else {
+		commentStr = fmt.Sprintf("%s %s in %s ", inst.cmdName, srcHash, inst.dest)
```

Comment:
I should have known this comment was coming! :) Even though I factored it out further per your comments on prior PRs for this, there is more trimming possible for sure :)